src/overlays/actors/ovl_Bg_Keikoku_Saku/z_bg_keikoku_saku.h

```diff
@@ -3,17 +3,18 @@
 
 #include "global.h"
 
+#define BGKEIKOKUSAKU_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
+
 struct BgKeikokuSaku;
 
 typedef void (*BgKeikokuSakuActionFunc)(struct BgKeikokuSaku*, GlobalContext*);
 
 typedef struct BgKeikokuSaku {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x18];
-    /* 0x015C */ BgKeikokuSakuActionFunc actionFunc;
-    /* 0x0160 */ char unk_160[0x4];
-} BgKeikokuSaku; // size = 0x164
-
+    /* 0x000 */ DynaPolyActor Dyna;
```

Comment:
```suggestion
    /* 0x000 */ DynaPolyActor dyna;
```
lowercase for struct member names.